lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -85,6 +90,39 @@ export class PublicClientApplication extends ClientApplication implements IPubli
         if (!account) {
             throw BrowserAuthError.createNoAccountError();
         }
+        const thumbprint: RequestThumbprint = {
+            clientId: this.config.auth.clientId,
+            authority: request.authority || "",
+            scopes: request.scopes,
+            homeAccountIdentifier: account.homeAccountId
+        };
+        const silentRequestKey = JSON.stringify(thumbprint);
+        let response = this.activeSilentTokenRequests.get(silentRequestKey);
```

Comment:
Might be good to not reassign the variable here. could do `cachedResponse` here and just response inside the `if` statement.

Reply:
Done.